pom.xml

```diff
@@ -28,7 +28,7 @@
     <version.mybatis>3.5.15</version.mybatis>
     <version.joda-time>2.12.5</version.joda-time>
     <version.uuid-generator>4.3.0</version.uuid-generator>
-    <version.feel-scala>1.17.5</version.feel-scala>
+    <version.feel-scala>1.18.1</version.feel-scala>
```

Comment:
❓ Since I see the backport labels here, should we really update all maintenance versions to a new FEEL Engine *minor* version? Instead, I would propose updating master to the newest minor version and maintenance versions to respective patch levels.

Reply:
I wouldn't dare introducing regressions via a patch release, given this list:
https://github.com/camunda/feel-scala/releases/tag/1.18.0

Reply:
I removed the backport labels. Will move to [1.17.9](https://github.com/camunda/feel-scala/releases/tag/1.17.9) for the backports.